src/services/web3/web3.plugin.ts

```diff
@@ -89,10 +89,7 @@ export default {
     });
 
     const chainId = computed(() => {
-      if (pluginState.connector) {
-        return pluginState.connector.chainId;
-      }
-      return Number(process.env.VUE_APP_NETWORK);
+      return pluginState.connector?.chainId;
```

Comment:
I think we should explicitly return the user's chainId here (or null if not connected) and not mix it up with the app chainId. wdyt @1saf?

Reply:
I think the appChainId return here was a guard for when user chain id here was undefined in some cases

Reply:
Yeah that's my point, in those cases we should be either handling that it's undefined or using the app network chainId. Since I've fixed all the other references, I think it's less likely to be a problem. Will need to thoroughly test the previews.